run-tests.sh

```diff
@@ -23,7 +23,7 @@ trap cleanup EXIT
 
 python -m check_manifest
 python -m sphinx.cmd.build -qnNW docs docs/_build/html
-eval "$(docker-services-cli up --search ${SEARCH:-opensearch} --env)"
+eval "$(docker-services-cli up --search ${SEARCH:-opensearch2} --env)"
 python -m pytest --runpytest=subprocess
```

Comment:
nit: that change was not necessary or?

Reply:
Opensearch 1 is about to be out of support in 4 weeks and we use opensearch2 everywhere else. Just making it better while I am there.